### PR TITLE
fixes github embeds pushing chat bodies out of viewport

### PR DESCRIPTION
**Pull request by barisusakli**

### CODE REVIEW

**Comment by barisusakli** [comment]
before
image


after

image

File: less/default.less [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .github-embeds-container {
 	margin-top: 1rem;
+	margin-left: 0px;
+        margin-right: 0px;
```
Comment (julianlam) [inline less/default.less:4]: `Mixed spaces and tabs. eslint(no-mixed-spaces-and-tabs)`

File: less/default.less [hunk 2]
```diff
@@ -42,6 +44,8 @@
 		background: none;
 		border: 0px;
 		margin-top: 1rem;
+		white-space: break-spaces;
+    		word-break: break-word;
```
Comment (julianlam) [inline less/default.less:48]: `Mixed spaces and tabs. eslint(no-mixed-spaces-and-tabs)`

**Review by julianlam — CHANGES_REQUESTED** [review]
:trollface:

**Comment by julianlam** [comment]
v0.7.8

**Pull request merged**